app/scripts/views/coppa/coppa-age-input.js

```diff
@@ -56,6 +56,16 @@ define(function (require, exports, module) {
       age.val(age.val().substr(0, 3));
     },
 
+    onKeyDown: function (event) {
+      // force interger input and submit on enter
+      var x = event.which;
+      if ((x > 8 && x < 48) || (x > 57 && x < 96) || x > 105) {
```

Comment:
I think this could be made more clear if an explicit `isKeyADigit` was used.

I imagine it would go something like:
* Add digit keycodes to keycodes.js
* Create a helperFunction `isKeyADigit` or similar that returns `true` if the key is a digit and `false` otherwise.
* Call the helper, take appropriate action.

Then something along the lines of:
```js
if (! isKeyADigit(keyCode)) {
  event.preventDefault();
} else...
```